src/db/migrations/26-add-jagwar-twin-artifacts.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex';
+
+import { createArtistIdFromEthereumAddress } from '../../types/artist';
+import { TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatformType } from '../../types/platform';
+import { addNftFactory, removeNftFactory } from '../migration-helpers';
+
+const jagwarTwinAddress = '0xe80c7fee3d41e311e0351244231919e04e699e56';
+
+const ARTIFACTS: NftFactory = {
+  address: '0x84F89f7bAcb20970073Be33F22847e58fbe78992',
+  startingBlock: '15316663',
+  platformId: jagwarTwinAddress,
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: true,
+  approved: true,
```

Comment:
i think we should leave approved: true (for the contract to be approved) but make autoApprove false since we only want to approve specific nfts from this contract we know are songs.

also, i noticed that when indexing this contract, a bunch of errors show up (when i query the db with `select * from raw_nft_process_errors;` ) of `Error mapping track: TypeError: Cannot read properties of undefined (reading 'startsWith')` -> i'm sure some errors are expecting given the contract is messy, but would be good to get better error messages, which i think will be solved with some of the other comments i made on this PR

Reply:
Right, those processing errors relate to the non-audio NFTs on the contract. The latest approach eliminates that TypeError when we choose to autoApprove, but those processing errors remain nevertheless. 

Let's chat about whether to lean on upcoming approvals mechanisms or if we should explore per NFT overrrides to approve the audio contracts :)